packages/@atjson/offset-annotations/src/annotations/table-of-contents.ts

```diff
@@ -1,20 +1,20 @@
 import { BlockAnnotation } from "@atjson/document";
 
 /**
- * A table of content is designed to enhance the usability and navigation of
+ * Table of contents is designed to enhance the usability and navigation of
  * our content. This feature provides users with a structured and accessible
  * way to navigate through various sections of a story, especially long-form
  * articles, improving the overall user experience and engagement. It will
  * make it easier for users to find specific sections or information quickly.
  * It also encourages readers to explore more content by providing them easy
  * access to different parts of our content.
  *
- * Table of Content displays data as a list of links that tag to headers that
+ * Table of Contents displays data as a list of links that anchor to headers that
```

Comment:
This comment should also be updated. The Table of Contents block actually is not opinionated about whether its contents are a list of links. You might instead say that its contents are meant to contain navigation to other parts of the document (which may take the form of a list of anchor links, just as one example)